proto/oracle/v1/query.proto

```diff
@@ -157,16 +202,23 @@ message QueryRequestSearchResponse {
 // QueryRequestPriceRequest is request type for the Query/RequestPrice RPC
 // method.
 message QueryRequestPriceRequest {
+  // Symbol represents currency unit
```

Comment:
it's not just `currency`

Reply:
Fixed to `unit of data indicating what the data is`